projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/history/event-log/event-log-table.component.html

```diff
@@ -8,7 +8,7 @@
   </tr>
   </thead>
   <tbody>
-  <tr *ngFor="let event of events" (click)="select(event)" [ngClass]="{'EventLogTable-Selected': selected === event}" tabindex="0" attr.aria-label="{{getAriaLabelforRow(event)}}">
+  <tr *ngFor="let event of events" (click)="select(event)" (keyup)="select(event)" [ngClass]="{'EventLogTable-Selected': selected === event}" tabindex="0" attr.aria-label="{{getAriaLabelforRow(event)}}">
```

Comment:
same as above

Reply:
Yes, it fixes the issue raised by sonarCloud

Reply:
Sonar accessibility check doesn't like it if you don't provide a keyboard equivalent for a mouse action, quite correctly